plugins/modules/ilo_redfish_config.py

```diff
@@ -60,8 +66,150 @@
     description:
       - Value of the attribute to be configured.
     type: str
+  raid_details:
+    required: false
+    description:
+      - List of RAID details that need to be configured in the given server
```

Comment:
```suggestion
      - List of RAID details that need to be configured in the given server.
```
Description paragraphs should always consist of complete sentences.